otx/algorithms/classification/adapters/mmcls/data/datasets.py

```diff
@@ -149,6 +149,9 @@ def evaluate(
             self.class_acc = True
 
         eval_results = super().evaluate(results, metrics, metric_options, logger=logger)
+        for k in metric_options["topk"]:
+            eval_results[f"accuracy_top-{k}"] /= 100
+            assert 0 <= eval_results[f"accuracy_top-{k}"] <= 1
```

Comment:
How about using `logger.warning` instead of `assert`? If `accuracy-top-1` is given as `nan`, otx will die.

Reply:
is `nan` metric expected? I think it is not normal case if we have nan in metric

Reply:
Yes, as you said, `nan` is not expected for normal situation. But I believe even though `nan` is occurred in training, otx should be alive. 
How about `raise`? In my memory, we agreed to avoid `assert` in our product. @harimkang @sungmanc Could you share your thoughts?

Reply:
I think we don't need this check at all. It is useless. If the accuracy come from mmcls in the range from [0,1] we wouldn't catch it any way.